webservices/partition/sched_b.py

```diff
@@ -51,7 +51,7 @@ def index_factory(cls, child):
         return [
             sa.Index(None, c.rpt_yr),
```

Comment:
I can't speak for the need to index on this column, it did carry over to the new vsum tables, if it's not being sorted on this column or being referenced I don't see any need to index it.